prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md

```diff
@@ -0,0 +1,41 @@
+# prep-html-skeleton-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What tags are necessary for a complete HTML Skeleton?
+
+- What type of content belongs within the `<head>` of an HTML document?
+
+- What type of content belongs within the `<body>` of an HTML document?
+
+- Where must the `DOCTYPE` declaration appear in a valid HTML document?
+
```

Comment:
Don't forget about the quiz questions!

Reply:
Here are the answers:
1- What tags are necessary for a complete HTML Skeleton?
<!DOCTYPE html>
<html>
<head> <title> </title> </head>
<body></body>
</html>
2- What type of content belongs within the '<head>' of an HTML document?
It contains information about the document which includes metadata (character set, title, styles (CSS), scripts).
3- What type of content belongs within the `<body>` of an HTML document?
It contains all the contents of an HTML document which is Structured content(headings, paragraphs, lists), Embedded content(images, videos, audio, iframes), and forms to collect data from users.
4- Where must the `DOCTYPE` declaration appear in a valid HTML document?
It must be the very first thing in the HTML document, before the <html> tag.